apps/docs/pages/guides/cli/getting-started.mdx

```diff
@@ -75,6 +60,27 @@ and run one of the following:
 - `sudo dpkg -i <...>.deb`
 - `sudo rpm -i <...>.rpm`
 
+</TabPanel>
+<TabPanel id="npx" label="npm / Bun">
+
+Run the CLI by prefixing each command with `npx`:
+
+```sh
+npx supabase <command>
```

Comment:
not sure why the `d` has a different styling

<img width="76" alt="Screenshot 2023-12-12 at 14 44 17" src="https://github.com/supabase/supabase/assets/25671831/d2b9a2e4-b661-4d82-9c25-f077e3310725">

Reply:
the `bash` formatting doesn't like <> unfortunately